try {
    console.log(a);
} catch(e) {
    console.log(e);
}

console.log('continue');
